scipy/integrate/_ivp/common.py

```diff
@@ -8,10 +8,12 @@
 EPS = np.finfo(float).eps
 
 
-def validate_first_step(first_step):
+def validate_first_step(first_step, t_bound, t0):
```

Comment:
This is really nitcpicky, so feel free to ignore, but I would order the parameters like this: `(first_step, t0, t_bound)`